integrations/azure-devops/azure_devops/client/azure_devops_client.py

```diff
@@ -218,11 +221,12 @@ async def _fetch_work_items_in_batches(
             )
             work_items_response.raise_for_status()
             work_items.extend(work_items_response.json()["value"])
+
         return work_items
 
     def _add_project_id_to_work_items(
-        self, work_items: list[dict[str, Any]], project_id: str
-    ) -> None:
+        self, work_items: list[dict[str, Any]], project_id: str, project: dict[str, Any]
+    ) -> list[dict[str, Any]]:
```

Comment:
rename the function to __add_project_details_to_work_items(self, work_item, project)

then you can do 
__projectId = project["id]
 __project = project

Reply:
Done